types/jenkins/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
To use FormData type definition, add "dom".

Reply:
I don't think that adding `DOM` to a package named `node-jenkis` is a good thing. Do you mean to import `form-data` or something?

Reply:
I mean that there's a library with that name, but maybe my memory is bad. In any case, I'm pretty sure I have read code which is for node that has this type without DOM, e.g. node-fetch.

Reply:
I understand what you mean. I'll take a look at libraries like node-fetch and try to create form-data type definition without using DOM.

Reply:
updated FormData type definition to import `form-data` library.
[`371a7a1` (#66784)](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66784/commits/371a7a16d4429d15cb19afc37097165c404a9864)

Reply:
Thanks, looks correct given upstream has this same version range in its package.json.